### PR TITLE
Absorbent galoshes.

### PR DESCRIPTION
**Pull request by xxalpha**
These shoes will dry floors if they are wet with water, but not lube.

Also a new chemical, drying agent. Spill this on galoshes to make absorbent galoshes.
dryshoes

### CODE REVIEW

**Comment by WJohn** [comment]
How do you get access to these? Admin only as of right now, I presume?

File: code/modules/reagents/Chemistry-Recipes/Others.dm [hunk 1]
```diff
@@ -12,14 +12,14 @@
 	result = "lube"
 	required_reagents = list("water" = 1, "silicon" = 1, "oxygen" = 1)
 	result_amount = 4
-	
+
 /datum/chemical_reaction/spraytan
 	name = "Spray Tan"
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: is this even used?

Comment (Miauw62) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: its better than most of the reagents you made

**Comment by Miauw62** [comment]
why do people think adding things to make janitor less slippery will make it better.
what you need to add is a mop that lets you mop the floor with lube.

**Comment by JJRcop** [comment]
I've always wanted to be able to wring out the mop and use a dry mop to dry up the floor.

File: code/modules/reagents/Chemistry-Recipes/Others.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: ill fucking fight you

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -62,6 +62,16 @@
...
 
+/obj/item/clothing/shoes/galoshes/dry
+	name = "absorbent galoshes"
+	desc = "A pair of orange rubber boots, designed to prevent slipping on wet surfaces while also drying them."
+	icon_state = "galoshes_dry"
+
+/obj/item/clothing/shoes/galoshes/dry/step_action()
+	var/turf/simulated/t_loc = get_turf(src)
+	if(istype(t_loc) && t_loc.wet)
+		t_loc.MakeDry(1) //dry if water, but not lube
```
Comment (RemieRichards) [inline code/modules/clothing/shoes/miscellaneous.dm]: use the define you added, instead of 1, then remove the comment.

**Comment by RemieRichards** [comment]
@WJohn make the chem and splash it on your galoshes.

**Comment by Lumipharon** [comment]
Not being able to dry floors with a mop is pretty retarded. That is 50% of what a mop is used for.

**Comment by WJohn** [comment]
Are we really adding a new chemical whose ENTIRE purpose is to improve the shoes of the janitor one time? Tell me they can be applied to other shoes or SOMETHING.

**Comment by RemieRichards** [comment]
@WJohn the chem also absorbs the floor on it's own, so you can put it in something and shoot it on floors to absorb the chemicals? it's a bit odd but yeah

**Comment by Iamgoofball** [comment]
@WJohn It works as reverse water/spacelube.

Janitor can fill up his backpack with this + water and clean a hallway then dry up the water immediately.

**Comment by Supermichael777** [comment]
so basicaly it can b mixed with water in a backpack to produce gheto spacecleaner

**Comment by newfren** [comment]
Why would you be cleaning with water if you weren't intentionally slipping people I don't understand.

**Comment by Razharas** [comment]
Having any chem applying galoshes instead of just dry would be much better
Other than this pr just needs conflict fix and we are ready to go
@xxalpha

**Comment by xxalpha** [comment]
Fixed conflicts

**Pull request merged**